src/main/kotlin/no/nav/familie/ba/mottak/task/OpprettBehandleSakOppgaveTask.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.familie.ba.mottak.task
+
+import no.nav.familie.ba.mottak.integrasjoner.JournalpostClient
+import no.nav.familie.ba.mottak.integrasjoner.Journalstatus
+import no.nav.familie.ba.mottak.integrasjoner.OppgaveClient
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+@TaskStepBeskrivelse(taskStepType = OpprettBehandleSakOppgaveTask.TASK_STEP_TYPE, beskrivelse = "Opprett \"BehandleSak\"-oppgave")
+class OpprettBehandleSakOppgaveTask(private val journalpostClient: JournalpostClient,
+                                    private val oppgaveClient: OppgaveClient,
+                                    private val taskRepository: TaskRepository) : AsyncTaskStep {
+
+    val log: Logger = LoggerFactory.getLogger(OpprettBehandleSakOppgaveTask::class.java)
+
+    override fun doTask(task: Task) {
```

Comment:
Vet ikke om vi har diskutert flyten godt nok, men for å kunne lage disse oppgavene, så må journalposten være ferdigstilt, og man må ha saksnummer oppdatert.

Reply:
Har tatt utgangspunkt i at det er utført før vi kommer til dette steget, da jeg tenkte at det passet bedre som en egen task, men ser at det blir feil å sjekke på status MOTTATT under. Det skal jeg rette opp i 👍